protocol/0078-NWLI-network_wide_limits.md

```diff
@@ -0,0 +1,128 @@
+# Network wide limits
+
+This spec describes network wide limits aimed at keeping the overall system performant and responsive.
+Vega has been designed with low-latency in mind so that the responsiveness of the network doesn't unduly impact the trading experience. Furthermore, the current implementation relies on an interplay between the execution layer (`core`) and the data consumption layer (`data node`). For these reasons limits allowing to exercise a degree of control over the computational and data generation loads need to be put in place. These limits need to be adjustable so that each network/shard can be setup for maximum accessibility (hardware constraints permitting) while preserving the desirable user experience and system stability, and mitigating the possibility of a malicious actor trying to deliberately disrupt the system by flooding it with superfluous requests.
+
+## Number of markets
+
+Introduce a new network parameter `limits.markets.max` controlling the maximum number of markets allowed within a network.\
+Default value: `50`.
+
+Each market in a `Pending` [state](./0043-MKTL-market_lifecycle.md) should increment the **total market count** by 1. If a market reaches a `Rejected`, `Cancelled`, `Closed` or `Settled` state the total market count should be decremented by 1.
+As soon as the total market count reaches the value specified by `limits.markets.max` parmeter no other further market proposals should be accepted. Any markets in a `Proposed` state at that stage should get rejected. If the total market count drops below `limits.markets.max` the network should start accepting market proposals again.
+
+### Change of network parameter
+
+If the network parameter gets increased via a governance vote no further actions are needed.\
+If it gets decreased below the current total market count then no further actions are needed. Specifically: all existing markets continue functioning in their respective states, but no further proposals are accepted until the count drops below the new limit.
+
+## Parties in a market
+
+Introduce a new network parameter `limits.markets.maxParties` controlling the maximum number of parties allowed within any given market. Default value: `100,000`.
+
+A party gets counted towards the limit if it has either open orders or open positions in the market. Once party has no open orders and no open positions it gets removed from a **total party count** within the market. Once the limit gets reached the market accepts no further orders from parties that are not already in the market.
+Once the total party count drops below the limit market accepts orders from any party again (provided its [state](./0043-MKTL-market_lifecycle.md#market-lifecycle-statuses) allows that).
+
+The limit does not apply to liquidity providers.
```

Comment:
I'm not sure I understand? Does that mean that a party which is a liquidity order doesn't take part of this limit? Or that if this limit is reached, and an LP want to join AND the max LP limit is not reached, then the new lp submission should be accepted?